portfolio/src/main/java/com/budiwebsite/model/Comment.java

```diff
@@ -0,0 +1,20 @@
+package com.budiwebsite.model;
+
+public class Comment {
```

Comment:
Add some documentation to the class explaining what it stores and when/how it's expected to be used